### PR TITLE
{bio} [dummy] WEKA-3.6.12-Java-1.7.0_67.eb (REVIEW)

### PR DESCRIPTION
**Pull request by pescobar**

### CODE REVIEW

File: easybuild/easyconfigs/w/WEKA/WEKA-3.6.12-Java-1.7.0_67.eb [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics
+# Biozentrum - University of Basel
+
+easyblock = "Tarball"
+
+name = 'WEKA'
+version = '3.6.12'
+versionsuffix = '-Java-1.7.0_67'
```
Comment (fgeorgatos) [inline easybuild/easyconfigs/w/WEKA/WEKA-3.6.12-Java-1.7.0_67.eb]: imho, you rely on too many java versions ;-) 1.7.0_80 would be great here

File: easybuild/easyconfigs/w/WEKA/WEKA-3.6.12-Java-1.7.0_67.eb [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+ Weka contains tools for data pre-processing, classification, regression, clustering, 
+ association rules, and visualization. It is also well-suited for developing new machine 
+ learning schemes."""
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+source_urls = ['http://prdownloads.sourceforge.net/weka/']
+sources = ['%s-%s.zip' % (name.lower(), version.replace('.','-'))]
+
+dependencies = [('Java', '1.7.0_67')]
```
Comment (fgeorgatos) [inline easybuild/easyconfigs/w/WEKA/WEKA-3.6.12-Java-1.7.0_67.eb]: ditto

File: easybuild/easyconfigs/w/WEKA/WEKA-3.6.12-Java-1.7.0_67.eb [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+sanity_check_paths = {
+    'files': ["weka.jar"],
+    'dirs': [""]
+}
+
+modextravars = {
+    'WEKAINSTALL': '$root',
+}
+
+modloadmsg = " execute WEKA: java -jar \\$EBROOTWEKA/weka.jar "
```
Comment (fgeorgatos) [inline easybuild/easyconfigs/w/WEKA/WEKA-3.6.12-Java-1.7.0_67.eb]: interesting, we could generalise upon this idea ;-)

**Comment by boegel** [comment]
Jenkins: test this please

File: easybuild/easyconfigs/w/WEKA/WEKA-3.6.12-Java-1.7.0_67.eb (hunk 1 above)
Comment (boegel) [inline easybuild/easyconfigs/w/WEKA/WEKA-3.6.12-Java-1.7.0_67.eb]: I suspect Jenkins is complaining about this missing Java version, so yeah, let's bump it?

**Comment by wpoely86** [comment]
lgtm

File: easybuild/easyconfigs/e/ELPH/ELPH-1.0.1-ictce-5.3.0.eb [hunk 4]
```diff
@@ -0,0 +1,36 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics
+# Biozentrum - University of Basel
+
+easyblock='MakeCp'
```
Comment (boegel) [inline easybuild/easyconfigs/e/ELPH/ELPH-1.0.1-ictce-5.3.0.eb]: spaces around `=`

File: easybuild/easyconfigs/g/GLIMMER/GLIMMER-3.02b-ictce-5.3.0.eb [hunk 5]
```diff
@@ -20,13 +21,27 @@ toolchain = {'name': 'ictce', 'version': '5.3.0'}
...
     'files': ["bin/glimmer3", "glim302notes.pdf", "LICENSE"],
     'dirs': ["docs", "lib", "sample-run", "scripts"]
 }
 
+# fix hardcoded path to scripts folder
+postinstallcmds = ["sed -i -e 's|/fs/szgenefinding/Glimmer3/scripts|%(installdir)s/scripts|' %(installdir)s/scripts/*"]
+# fix hardcoded path to bin folder
+postinstallcmds += ["sed -i -e 's|/fs/szgenefinding/Glimmer3/bin|%(installdir)s/bin|' %(installdir)s/scripts/*"]
+# fix hardcoded path to elph binary
+postinstallcmds += ["sed -i -e 's|/nfshomes/adelcher/bin/elph|${EBROOTELPH}/bin/elph|' %(installdir)s/scripts/*"]
```
Comment (boegel) [inline easybuild/easyconfigs/g/GLIMMER/GLIMMER-3.02b-ictce-5.3.0.eb]: not a big fan of this, but I don't have a really good alternative, at least not until https://github.com/hpcugent/easybuild-framework/issues/1400 is fixed

File: easybuild/easyconfigs/w/WEKA/WEKA-3.6.12-Java-1.7.0_80.eb [hunk 6]
```diff
@@ -0,0 +1,37 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics
+# Biozentrum - University of Basel
+
+easyblock = "Tarball"
+
+name = 'WEKA'
+version = '3.6.12'
+versionsuffix = '-Java-1.7.0_80'
```
Comment (boegel) [inline easybuild/easyconfigs/w/WEKA/WEKA-3.6.12-Java-1.7.0_80.eb]: avoiding hardcoding `1.7.10_80` in two places

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/f510f688fda6f7f33444 for a full test report.

**Comment by pescobar** [comment]
replaced by this https://github.com/hpcugent/easybuild-easyconfigs/pull/1986

closing

**Comment by pescobar** [comment]
closing

**Comment by wpoely86** [comment]
Test report by @wpoely86
**SUCCESS**
Build succeeded for 5 out of 5 (4 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/1c83e38f852b4dfe9c53 for a full test report.

**Comment by rjeschmi** [comment]
Test report by @rjeschmi
**SUCCESS**
Build succeeded for 0 out of 0 (1 easyconfigs in this PR)
Linux centos 6.6, AMD Opteron(tm) Processor 6380, Python 2.6.6
See https://gist.github.com/18b5ab8baebf8f2fbafe for a full test report.

**Comment by rjeschmi** [comment]
Test report by @rjeschmi
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux centos 6.6, AMD Opteron(tm) Processor 6380, Python 2.6.6
See https://gist.github.com/232e6b6b575e6546ba9a for a full test report.

**Pull request closed without merging**